server/src/main/java/org/opensearch/indices/replication/checkpoint/PublishCheckpointAction.java

```diff
@@ -146,12 +146,11 @@ public void handleResponse(ReplicationResponse response) {
                                 timer.time()
                             )
                         );
-                        task.setPhase("finished");
```

Comment:
Why this `setPhase` call is removed ? If it is not useful, let's remove same during failure as well (on line 156)

Reply:
Not intentional, thanks for catching this, it is useful in tracking state in the `ReplicationTask`.